mats/misc.ms

```diff
@@ -3483,8 +3483,8 @@
 )
 
 (mat sci-bug
-  (equal? (expt 10.0 (- 21)) 1e-21)
-  (equal? (flexpt 10.0 (- 21.0)) 1e-21)
```

Comment:
It seems that this check of pow ought to be precise on all operating systems. See my comment on mats/5_3.ms.